Makefile

```diff
@@ -156,10 +156,11 @@ lint: node_modules/.installed
 	mypy --ignore-missing-imports    \
 		pyodide-build/pyodide_build/ \
 		src/ 					     \
-		packages/micropip/src/ 		 \
 		packages/*/test* 			 \
 		conftest.py 				 \
 		docs
+	mypy --ignore-missing-imports    \
+		packages/micropip/src/
```

Comment:
What was the issue there? Passing everything should give it more type information I imagine.
If you don't want to debug that here, maybe add a TODO in a comment?

Reply:
mypy gets angry about there being multiple files called `setup.py`:
https://github.com/python/mypy/issues/10428
I tried a bunch of different things to fix it but this was the only thing that seemed to work well. Will add a comment, hopefully mypy will improve this eventually.